Sources/Rendering/Core/ImageSlice/index.js

```diff
@@ -101,6 +101,11 @@ function vtkImageSlice(publicAPI, model) {
   };
 
   publicAPI.getBoundsForSlice = (slice, thickness) => {
+    if (!model.mapper?.getBoundsForSlice) {
+      vtkErrorMacro(
+        `Currently set mapper does not define method 'getBoundsForSlice()'`
+      );
+    }
```

Comment:
What about adding a getBoundsForSlice impl to ImageResliceMapper that calls vtkErrorMacro + returns uninitialized/invalid bounds?

Reply:
Or should that be in the `AbstractImageMapper` basically saying that derived classes should implement it?

Reply:
Ah yes, that's a good idea. Let's do that.